std/typecons.d

```diff
@@ -7143,3 +7143,265 @@ unittest
         ubyte, ubyte, T3, T3,
     );
 }
+
+/**
+ * Get the value of a field or property member. The member's name
+ * is resolved at runtime, rather than at compile-time.
+ *
+ * Note that currently only `pure @safe` property methods are supported.
```

Comment:
See also this failed forum thread I started a while ago: http://forum.dlang.org/post/rqydvgscbrjkrrvkbsnj@forum.dlang.org